src/main/resources/templates/aeolus/java/plain_gradle.yaml

```diff
@@ -0,0 +1,27 @@
+api: v0.0.1
+metadata:
+    name: java-gradle
+    id: java-windfile
+    description: Build Plan for a java Exercise
+    author: Andreas Resch
+    docker:
+        image: ls1tum/artemis-maven-template
+        tag: java17-20
+        volumes:
+            - ${WORKDIR}:${WORKDIR}
+            - ${TMPDIR}:${TMPDIR}
+        parameters:
+            - --cpus
+            - '"2"'
+            - --memory
+            - '"2g"'
+            - --memory-swap
+            - '"2g"'
+            - --pids-limit
+            - '"1000"'
```

Comment:
PR #5971 allowed admins to override these values. Will this no longer work with the new approach?

Reply:
it will be possible also for users to override those settings, however since this PR does not do that I removed it.